### PR TITLE
OSDOCS-12559: Add Known Issue

### PR DESCRIPTION
**Pull request by fmcdonal**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-12559

Link to docs preview:
https://84483--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_whats_new/osd-whats-new.html
https://84483--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_release_notes/rosa-release-notes.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Review by arendej — APPROVED** [review]
LGTM

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: osd_whats_new/osd-whats-new.adoc [hunk 1]
```diff
@@ -62,3 +62,7 @@ For more information on region availabilities, see xref:../osd_architecture/osd_
...
+
+[id="osd-known-issues_{context}"]
+== Known issues
+* {OCP} 4.14 introduces an updated HAProxy image from 2.2 to 2.6. This update introduces a change in behavior enforcing strict RFC 7230 compliance, rejecting requests with multiple `Transfer-Encoding` headers. This may cause exposed pods in {product-title} 4.14 clusters sending multiple `Transfer-Encoding` headers to respond with a `502 Bad Gateway` or `400 Bad Request error`. To avoid this issue, ensure that your applications are not sending multiple `Transfer-Encoding` headers. For more information, see link:https://access.redhat.com/solutions/7055002[Red Hat Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-43095[*OCPBUGS-43095*])
```
Comment (mletalie) [inline osd_whats_new/osd-whats-new.adoc:68]: I see that you added "To avoid this issue..." and comma we discussed. Thanks!

**Review by mletalie — COMMENTED** [review]
/label peer-review-done

File: osd_whats_new/osd-whats-new.adoc [hunk 2]
```diff
@@ -62,3 +62,7 @@ For more information on region availabilities, see xref:../osd_architecture/osd_
...
+
+[id="osd-known-issues_{context}"]
+== Known issues
+* OpenShift Container Platform 4.14 introduces an updated HAProxy image from 2.2 to 2.6. This update introduces a change in behavior enforcing strict RFC 7230 compliance, rejecting requests with multiple `Transfer-Encoding`` headers. This may cause exposed pods in {product-title} 4.14 clusters sending multiple `Transfer-Encoding` headers to respond with a `502 Bad Gateway` or `400 Bad Request error`. Ensure that your applications are not sending multiple `Transfer-Encoding` headers. For more information see link:https://access.redhat.com/solutions/7055002[Red Hat Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-43095[*OCPBUGS-43095*])
```
[the remark this answers is not in the document]
Comment (fmcdonal) [reply osd_whats_new/osd-whats-new.adoc]: done thanks

[the remark this answers is not in the document]
Comment (fmcdonal) [reply osd_whats_new/osd-whats-new.adoc]: @arendej will I change to "introduced"

File: osd_whats_new/osd-whats-new.adoc (hunk 1 above)
Comment (fmcdonal) [reply osd_whats_new/osd-whats-new.adoc:68]: done thanks

**Pull request merged**